cogs/rcon.py

```diff
@@ -0,0 +1,262 @@
+#un implemented.
+
+import discord
+from discord.ext.commands import Cog, command, has_any_role
+from utils import load_config
+import a2s
+import re
+from rcon import Client
+#https://pypi.org/project/rcon/
+
+
+
+global_config = load_config()
+config = global_config
+
+class rcon(Cog):
+
+    #test command
+    @command()
+    async def test(self, ctx):
+        await ctx.trigger_typing()
+        await ctx.send("test")
+    
+    #list all servers
+    @command()
+    @has_any_role('Staff', 'Server Mods', 'Senior Staff', 'Fub')
+    async def servers(self, ctx):
```

Comment:
wont this command conflict with other server commands we have? 

we might wanna think about namespacing our commands as a whole better than we do now

Reply:
Removed this command, this was more for testing and I forgot to remove it, my bad. It is replaced by just doing !server which the spits out a help menu stating what args can be used. **!server us stop**. !servers checks for an active imp. So it might be confusing. I can easily change it
![serverargs](https://user-images.githubusercontent.com/44575567/141369386-91d0b999-8a69-402e-b603-5cc2fec32f4a.PNG)
.